hawkular-alerters/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.hawkular.alerts</groupId>
+    <artifactId>hawkular-alerts</artifactId>
+    <version>1.6.0.Final-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>hawkular-alerters</artifactId>
+  <packaging>pom</packaging>
+
+  <name>Hawkular Alerting: Alerters parent</name>
+  <description>Parent module of alerter artifacts</description>
+
+  <profiles>
+    <profile>
+      <id>default</id>
+      <modules>
+        <module>hawkular-elasticsearch-alerter</module>
+      </modules>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+    </profile>
+
+    <profile>
+      <id>openshift</id>
+      <modules>
+        <module>hawkular-elasticsearch-alerter</module>
+      </modules>
+    </profile>
+  </profiles>
+
+</project>
```

Comment:
I think alerters/externsions/plugins should be under the hAlerts repository unless there is a good reason to split them.
Metrics alerter is good to have it on the metrics repo because it introduces complexity on dependencies, and perhaps the semantic and operations are more related to the metrics context rather than alerting.
But on this alerter (and future ones) I think that they are like plugins, so, some library dependency is expected but logic is generic enough to maintain them here.

It is a good point to think if this require a separate lifecycle, well, I expect once is addressed and more mature it will follow the regular lifecyce of alerting core or plugins.

We can revisit this in the future if we see that it has a different requeriments rate rather than alerting, but in any case, as it has to be packaged on a higher metrics.ear, we will have same issues, so I don't see a worth simplification of the process.